### PR TITLE
5.2 release notes

### LINKED ISSUE #7358: 5.2 Release Notes
For a release we need release notes.

### PR DESCRIPTION
**Pull request by djbrooke**
**What this PR does / why we need it**:

It's the release notes! 

**Which issue(s) this PR closes**:

Closes #7358 

**Special notes for your reviewer**:

I didn't adjust any formatting, but please check the format especially in the installation section at the bottom.

**Suggestions on how to test this**:

Your favorite text editor! 

**Does this PR introduce a user interface change? If mockups are available, please link/include them here**:

N/A

**Is there a release notes update needed for this change?**:

N/A

**Additional documentation**:

N/A

### CODE REVIEW

**Review by pdurbin — COMMENTED** [review]
The content is fine. My review contains mostly minor suggestions and thoughts for the future.

**Comment by poikilotherm** [comment]
Chances we'll see #7048 in this? (Just sayin' because of those other necessary DB changes...)

File: doc/release-notes/5.2-release-notes.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+If this is a new installation, please see our [Installation Guide](https://guides.dataverse.org/en/5.2/installation/)
+
+## Upgrade Instructions
+
+0. These instructions assume that you've already successfully upgraded from Dataverse 4.x to  Dataverse 5 following the instructions in the [Dataverse 5 Release Notes](https://github.com/IQSS/dataverse/releases/tag/v5.0).
+
+1. Undeploy the previous version.
+
+`<payara install path>/bin/asadmin list-applications`
```
[the remark this answers is not in the document]
Comment (djbrooke) [reply doc/release-notes/5.2-release-notes.md]: added in https://github.com/IQSS/dataverse/pull/7390/commits/2ea51b235d041bd0bb5e9e2996a9f951e845f577

File: doc/release-notes/5.2-release-notes.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+- `service payara stop`
+- remove the generated directory: 
+`rm -rf <payara install path>/payara/domains/domain1/generated`
+- `service payara start`
+
+3. Deploy this version.
+`<payara install path>/bin/asadmin deploy dataverse-5.2.war`
+
+4. Restart payara
```
[the remark this answers is not in the document]
Comment (djbrooke) [reply doc/release-notes/5.2-release-notes.md]: Thanks - addressed the 2nd and 3rd items in https://github.com/IQSS/dataverse/pull/7390/commits/1880e8ddb591fb3b7839602b7ed38cbe84db5f84

[the remark this answers is not in the document]
Comment (djbrooke) [reply doc/release-notes/5.2-release-notes.md]: regarding the 1st bullet, good to hear it's OK for now but I'll be happy to update in the future !

**Comment by pdurbin** [comment]
> Chances we'll see #7048 in this? (Just sayin' because of those other necessary DB changes...)

It just went to QA so maybe! 😄 But the nice thing about a release train is that there's always another train. 🚂

**Pull request merged**